sdk/servicebus/Azure.Messaging.ServiceBus/tests/Sender/ServiceBusMessageBatchLiveTests.cs

```diff
@@ -54,8 +54,9 @@ async Task AddAndSendMessages()
                         if (batch.Count < 4500)
                         {
                             var diff = batch.MaxSizeInBytes - batch.SizeInBytes;
-                            // the difference in size from the max allowable size should be less than the size of 1 message
-                            Assert.IsTrue(diff < 220, diff.ToString());
```

Comment:
This just happened to work for instrumented messages before because the batch very close to the limit when attempting to add the last message. 

With the changes to AMQP conversion, the message size is now smaller as we don't include empty sections. This test started failing because the instrumented message size is 250 bytes.